Mailer/app/api/v1/Handlers/Mailers/MailLogsForEmailCountHandler.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Remp\MailerModule\Api\v1\Handlers\Mailers;
+
+use Remp\MailerModule\Api\JsonValidationTrait;
+use Remp\MailerModule\Api\v1\Handlers\Mailers\Traits\ParseLogFilterConditionsTrait;
+use Remp\MailerModule\Repository\LogsRepository;
+use Tomaj\NetteApi\Handlers\BaseHandler;
+use Tomaj\NetteApi\Params\InputParam;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+
+class MailLogsForEmailCountHandler extends BaseHandler
+{
+    private $logsRepository;
+
+    use JsonValidationTrait;
+    use ParseLogFilterConditionsTrait;
+
+    public function __construct(
+        LogsRepository $logsRepository
+    ) {
+        parent::__construct();
+        $this->logsRepository = $logsRepository;
+    }
+
+    public function params()
+    {
+        return [
+            new InputParam(InputParam::TYPE_POST_RAW, 'raw')
+        ];
+    }
+
+    public function handle($params)
+    {
+        $payload = $this->validateInput($params['raw'], __DIR__ . '/logs-count.schema.json');
+
+        if ($this->hasErrorResponse()) {
+            return $this->getErrorResponse();
+        }
+
+        $output = $this->logsRepository->allForEmail($payload['email']);
+
+        if (isset($payload['filter'])) {
+            $output->where($this->parseConditions($payload['filter']));
```

Comment:
Looking at our other APIs, please accept time filter parameters in `RFC3339` (`ISO 8601`) format **with time** so the client is responsible for specifying whole requested range.

Let's not assume client wants to work with full days. We can then get rid of the date kung-fu inside `parseConditions` method.